experimental/framework/fluid-static/src/fluidContainer.ts

```diff
@@ -0,0 +1,73 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { strict as assert } from "assert";
+import { EventEmitter } from "events";
+import { IAudience } from "@fluidframework/container-definitions";
+import { Container } from "@fluidframework/container-loader";
+import { IFluidLoadable } from "@fluidframework/core-interfaces";
+import {
+    LoadableObjectClass,
+} from "./types";
+import { RootDataObject } from "./rootDataObject";
+
+export class FluidContainer
+    extends EventEmitter
+    implements
+        Pick<Container, "audience" | "clientId" | "close" | "closed" | "connected" >,
+        Pick<RootDataObject, "initialObjects"> {
+    private readonly _container: Container | undefined = undefined;
+    private readonly _rootDataObject: RootDataObject | undefined = undefined;
+
+    public constructor(container: Container, dataObject: RootDataObject) {
```

Comment:
If we are using the static create functionality this should be private.

```suggestion
    private constructor(container: Container, dataObject: RootDataObject) {
```